allensdk/brain_observatory/behavior/data_objects/licks.py

```diff
@@ -122,3 +123,77 @@ def to_nwb(self, nwbfile: NWBFile) -> NWBFile:
         nwbfile.add_processing_module(licks_mod)
 
         return nwbfile
+
+
+class VBNLicks(Licks):
+
+    @classmethod
+    def from_json(cls, dict_repr: dict) -> "VBNLicks":
+        """Load Licks data for VBN from an input dictionary by loading a
+        stim and syncfile.
+
+        Parameters
+        ----------
+        dict_repr : dict
+            Dictionary of session parameters.
+
+        Returns
+        -------
+        vbn_licks : VBNLicks
+            Licks data file
+        """
+        behaveStim = BehaviorStimulusFile.from_json(dict_repr)
+        sync_file = SyncFile.from_json(dict_repr, permissive=True)
+        return cls.from_stimulus_file(sync_file, behaveStim)
+
+    @classmethod
+    def from_stimulus_file(
+            cls,
+            sync_file: SyncFile,
+            stimulus_file: BehaviorStimulusFile) -> "VBNLicks":
+        """Get lick data from pkl file.
+
+        This function assumes that the first sensor in the list of
```

Comment:
This looks like duplicated code. Please modify `Licks` instead with this new logic and delete this class.

Reply:
It's slightly different. The code loads the timing data directly from the sync file instead of the `StimulusTimestamps`. I seem to remember Scott talking to Corbett about this being the way it should be done for this sample (though I could be wrong). Looking through `StimulusTimestamps`, at least from the function names, it seems to have a lot of ophys specific code that we don't want and wouldn't run.

Reply:
Do we want to change the Licks class thusly? Are we sure there isn't some ophys specific running mode that crashes? Also, again I'm not sure that we can use the same class for these data as opposed to the ophys data. I guess I could make a function in Licks to use the combination stimfile and syncfile and call it something like `from_sync_file`